Update azure-arc-enabled-virtual-machines.md (#1901)
MicrosoftDocs/azure-stack-docs: azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md

Deploying Azure Arc Resource Bridge requires the following:

- The latest version of Azure CLI installed on all servers of the cluster.

>   - To install Azure CLI on each cluster node use RDP connection.

Reviewer: ```suggestion
  - To install Azure CLI on each cluster node, use RDP connection.
```

>   - Follow the instructions on this article: [Install Azure CLI](/cli/azure/install-azure-cli-windows).

Reviewer: ```suggestion
  - Follow the instructions in [Install Azure CLI](/cli/azure/install-azure-cli-windows).
```

>   - Follow the instructions on this KB: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows

Reviewer 2: This is an article, not a KB, and the link should be relative, i.e. /cli/azure/install-azure-cli-windows

Author: Done, replace KB by article and relative URI.

## Install PowerShell modules and update extensions

> To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps (through RDP or console session, remote Powershell is not supported):

Reviewer: ```suggestion
To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps (through RDP or console session; remote Powershell isn't supported):
```

> To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps (throught RDP or console session, remote Powershell is not supported):

Reviewer 2: Typo in this sentence, "throught" is not a word.

Author: Sorry, through vs throught.
